userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -193,13 +193,13 @@ Note that `maxUnavailable` cannot be higher than `maxSize`. Also, `maxUnavailabl
 This feature is only available for managed nodes.
 
 ## Updating managed nodegroups
-It is also possible to update specific fields of a managed nodegroup with the command `eksctl update nodegroup` without using `upgrade`.
 
-This new command currently only supports a few fields. It is intended to be used as a way to modify the configuration of a nodegroup without triggering an entire upgrade.
+`eksctl` allows updating the [UpdateConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-eks-nodegroup-updateconfig.html) section of a managed nodegroup.
+This section defines two fields. `MaxUnavailable` and `MaxUnavailablePercentage`. Your nodegroups are unaffected during
+the update, thus downtime shouldn't be expected.
 
-Right now, it will only update upgrade configuration.
-
-The command `update nodegroup` should be used with a config file using the `--config-file` flag. If the config file contains fields that cannot be updated through `eksctl update nodegroup`, a log will inform you of all the fields that remained unchanged.
+The command `update nodegroup` should be used with a config file using the `--config-file` flag. The nodegroup should
+contain an `nodeGroup.updateConfig` section. More on the field can be found [here](https://eksctl.io/usage/schema/#nodeGroups-updateConfig).
```

Comment:
But that's the wrong section. I see that it collapses, but that's because it's an open value. `taints` is the wrong section whoever is reading that with a screen reader will have a hard time understanding why we link to taints for updateConfig. :)

Reply:
as per our discussion, i am updating my suggestion 👍🏻 
```suggestion
The command `update nodegroup` should be used with a config file using the `--config-file` flag. The nodegroup should
contain an `nodeGroup.updateConfig` section. More information can be found [here](https://eksctl.io/usage/schema/#nodeGroups-updateConfig).
```